### PR TITLE
EndpointsBase

### PR DESCRIPTION
**Pull request by ciscoflores**
This PR:

**Add:**
-The EndpointsBase file which contains a struct with all the base endpoints url, in order to have a unique file with this information

### CODE REVIEW

File: Solidus-Swift-SDK/EndpointsBase.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//  Solidus-Swift-SDK
+//
+//  Created by Francisco Flores on 5/3/16.
+//  Copyright © 2016 Omar Guzman. All rights reserved.
+//
+
+import Foundation
+
+struct EndpointsBase {
+    static var products = "/api/products"
```
Comment (softr8) [inline Solidus-Swift-SDK/EndpointsBase.swift:12]: what if the backend is namespaced?

Comment (ciscoflores) [inline Solidus-Swift-SDK/EndpointsBase.swift:12]: @softr8 is only the base to construct the _"final"_ url request. I mean, those gonna be concatenated to the domain store, which could be _www.store.com_ or _https://store.com_. Also, this in order to have a cleaner and organised code.

Comment (softr8) [inline Solidus-Swift-SDK/EndpointsBase.swift:12]: ok, so, if solidus engine is mounted in a namespace, you will use `www.store.com/spree` ?

Comment (ciscoflores) [inline Solidus-Swift-SDK/EndpointsBase.swift:12]: If the _solidus_ engine is mounted in that _url_ the request is gonna be like _www.store.com/spree/api/products_

**Comment by softr8** [comment]
LGTM

**Pull request merged**